### DISCUSSION TITLE
Windows 11 ARM 64 retail version on Apple Silicon

### DISCUSSION THREAD

**Discussion by lcarson-2002** [utmapp/UTM#5319] [Windows]
Hi folks,

I have the Windows 11 ARM 64 Dev channel Insider Preview Release installed, as per the instructions. Since Windows 11 ARM 64 retail has been released has anyone tried upgrading or directly installing the retail version? My understanding is that it has the x64 compatibility software included in the retail release. Is there anything else required? I am wondering if the TPM compatibility is an issue regardless of the new Retail version - i.e. we still need the Insider Preview Release as it helps bypass requirements that UTM/qemu cannot meet?

**Comment by LostInCompilation** [comment]
> My understanding is that it has the x64 compatibility software included in the retail release.

What do you mean with that?
If you mean the x64 emulation layer to run x86_64 Apps under AArch64 Windows 11, that is included in the Insider Preview, too.

**Comment by lcarson-2002** [comment]
In the retail version they've included the x64 emulation layer to run x86_64 apps under AArch64 Windows 11; I had thought that was the main reason to use the Insider Preview Release because the emulation layer was only available there. What I do not know, is whether there are any other reasons keeping me from upgrading to the retail version. Honestly, I'm tired of the constant updates of the Dev version and constantly repairing/reinstalling Spice drivers, as well as the Spice guest agent.

**Comment by LostInCompilation** [comment]
You can switch your update channel in settings from Canary to something lower, to get way less updates.

**Comment by Pixel556** [comment]
VHDX soon will be required very importantly, because Microsoft is removing the registry requirement hack in the Canary channel.
